### ISSUE TITLE
secret data should be kept secret

### ISSUE THREAD

**Issue by sepulchered** [PeARSearch/PeARS#15]
It's a bad thing to store valuable data in repository e.g. `SECRET_KEY` from `config.py`.
Common practice is to use local config file that is imported in general config file and overwrite default options from general config. Or use system environment variables.

**Comment by stultus** [comment]
Agree.  But our first preference was to get the system working and we are still in pre-alpha, hence the SECRET_KEY and all are just throw away values.  We will be moving that to .env files during the upcoming refactoring.  Thanks for the heads up
